code/modules/mob/living/simple_animal/hostile/ordeal/white.dm

```diff
@@ -364,13 +365,13 @@
 
 	var/busy = FALSE
 	var/multislash_cooldown
-	var/multislash_cooldown_time = 5 SECONDS
+	var/multislash_cooldown_time = 4 SECONDS
 	var/multislash_damage = 75
 	var/multislash_range = 6
 	var/beam_cooldown
-	var/beam_cooldown_time = 15 SECONDS
+	var/beam_cooldown_time = 10 SECONDS
 	/// Red damage dealt on direct hit by the beam
-	var/beam_damage = 300
+	var/beam_damage = 400
```

Comment:
I don't think the beam needs both a 30% increase in damage AND a massive reduction in his post-fire stun.

Reply:
It has a terribly long pre-fire duration.

Reply:
> It has a terribly long pre-fire delay.

Yeah but you're buffing the effectiveness of it while obliterating it's punishment time. The wind-up can't reasonably be used to punish it... unless you're using a gun. Wiping out the cooldown after the beam (2 seconds to 0.5) means this guy has practically no time where he isn't either hitting you or blinking past you.